index.html

```diff
@@ -1,243 +1,361 @@
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"></a
+                  ><img src="./assets/images/vectors/M.svg" alt="" />
+                </li>
+                <li>
+                  <a href="#"
+                    ><img src="./assets/images/vectors/chill.svg" alt=""
+                  /></a>
                 </li>
               </ul>
-            </li>
-            <li class="skill-item">
-              <div><span>Frameworks</span></div>
-              <ul class="skill-set">
-                <li class="skill"></li>
-                <li class="skill"></li>
-                <li class="skill"></li>
-              </ul>
-            </li>
-            <li class="skill-item">
-              <div><span>Skills</span></div>
-              <ul class="skill-set">
-                <li class="skill"></li>
-                <li class="skill"></li>
-                <li class="skill"></li>
-              </ul>
-            </li>
-          </ul>
-        </article>
-      </section>
-
-      <section class="contact" id="contact">
-        <div class="contact_wrapper">
-          <h2 class="section-header">Contact me</h2>
-          <p class="description">If you have an application you are interested in developing,
-            a feature that you need built or a project that needs coding. I’d love to help with it </p>
-          <form action="https://formspree.io/f/xvoyerbw" method="post">
-            <input type="text" name="user_name" placeholder="Enter your name" required maxlength="30">
-            <input type="email" name="user_mail" placeholder="Enter your e-mail" required>
-            <textarea name="user_message" cols="30" rows="5" placeholder="Write your message here" required
-              maxlength="500"></textarea>
-            <button class="buttons" type="submit">Get in Touch</button>
-          </form>
-        </div>
-      </section>
-    </main>
-  </div>
-</body>
+            </div>
+            <button type="button" class="buttons">Get My Resume</button>
+          </article>
+          <article class="section-b skills">
+            <ul class="skills-list">
+              <li class="skill-item open">
+                <div><span>Languages</span></div>
+                <ul class="skill-set">
+                  <li class="skill">
+                    <figure>
+                      <img src="./assets/images/vectors/js.png" alt="" />
+                    </figure>
+                    <span>JavaScript</span>
+                  </li>
+                  <li class="skill">
+                    <figure>
+                      <img src="./assets/images/vectors/css.png" alt="" />
+                    </figure>
+                    <span>CSS</span>
+                  </li>
+                  <li class="skill">
+                    <figure>
+                      <img src="./assets/images/vectors/html.png" alt="" />
+                    </figure>
+                    <span>HTML</span>
```

Comment:
- Please ensure all images have `alt text`, and remember to include the link to the changes you have made in your report 🙏